libshaderc/README.md

```diff
@@ -0,0 +1,43 @@
+# libshaderc
+
+A library for compiling shader strings into SPIR-V.
+
+## Build Artifacts
+There are two main build artifacts that are created from a CMake build of
+libshaderc.
+The first is libshaderc which is a static library containing just the
+functionality needed. This has static dependencies on other libraries compiled
```

Comment:
a static library containing only the bits for libshaderc. It depends on other compilation targets: `glslang`, `OSDependent`, ...